flink-ml-lib/src/test/java/org/apache/flink/ml/classification/LogisticRegressionTest.java

```diff
@@ -253,6 +255,25 @@ public void testSaveLoadAndPredict() throws Exception {
                 logisticRegression.getRawPredictionCol());
     }
 
+    @Test
+    public void testSaveLoadServable() throws Exception {
+        LogisticRegression logisticRegression = new LogisticRegression().setWeightCol("weight");
+        LogisticRegressionModel model = logisticRegression.fit(binomialDataTable);
+
+        LogisticRegressionModelServable servable =
+                saveAndLoadServable(
+                        tEnv,
+                        model,
+                        tempFolder.newFolder().getAbsolutePath(),
+                        LogisticRegressionModel::loadServable);
+
+        assertEquals("features", servable.getFeaturesCol());
+        assertEquals("prediction", servable.getPredictionCol());
+        assertEquals("rawPrediction", servable.getRawPredictionCol());
+
+        assertArrayEquals(expectedCoefficient, servable.getCoefficient().values, 0.1);
```

Comment:
Should we verify the result of the `servable.transform(...)` similar to `PipelineTest#testPipelineModelServable`?

Reply:
I don't think we really need to verify the result of the `servable.transform(...)`, because it has been covered in `LogisticRegressionModelServableTest#testTransform`. If you are worrying about the consistency of test cases of `PipelineTest` and `LogisticRegressionModelServableTest`, I think we should change the `PipelineTest#testPipelineModelServable` to only verify the model data of each stages.

Reply:
`LogisticRegressionModelServableTest#testTransform` does not save/load servable.

We need to have a test that shows that we can save/load servable and use the loaded servable for online prediction, similar to why we have both `LogisticRegressionTest#testFitAndPredict` and `LogisticRegressionTest#testSaveLoadAndPredict`.